src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1086,6 +1086,9 @@ namespace WaterToAirHeatPumpSimple {
 								MixHumRat = OutAirFrac * FinalSysSizing( CurSysNum ).PrecoolHumRat + ( 1.0 - OutAirFrac ) * FinalSysSizing( CurSysNum ).RetHumRatAtCoolPeak;
 							}
 						}
+						// supply air condition is capped with that of mixed air to avoid SHR > 1.0
+						SupTemp = min( MixTemp, SupTemp );
+						SupHumRat = min( MixHumRat, SupHumRat );
 						OutTemp = FinalSysSizing( CurSysNum ).OutTempAtCoolPeak;
 						rhoair = PsyRhoAirFnPbTdbW( StdBaroPress, MixTemp, MixHumRat, RoutineName );
 						MixEnth = PsyHFnTdbW( MixTemp, MixHumRat );
```

Comment:
@Nigusse Sorry I didn't respond to this, but for reference, in my editor I have whitespace always shown.  That helps me see if it accidentally put spaces instead of tabs.  But, if the tab length is specified at 4, it still looks right.  GitHub by default uses a tab length of 8, which makes it immediately clear when lines simply don't line up anymore.